README.md

```diff
@@ -392,7 +392,7 @@ instance = SingleVMOpenEdXInstance.objects.create(
     sub_domain='dogwood.sandbox',
     name='Dogwood',
     fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+    branch_name='named-release/dogwood.rc',
```

Comment:
@antoviaque The code failed when it was calling https://api.github.com/repos/edx/edx-platform/git/refs/heads/named-release/dogwood and the result was an array of objects instead of a single object (github api returns an array of all branches that start with the given string if it can't find the branch with the exact name, which is kind of weird).

I just checked and see that our code also supports tags, you just have to initialize the instance with `ref_type='tags`' (`ref_type` defaults to `heads`).
Would you prefer I add `ref_type='tags'` and keep `branch_name='named-release/dogwood'` in the README?